scipy/special/basic.py

```diff
@@ -1394,15 +1388,14 @@ def lpmn(m, n, z):
     if (m < 0):
         mp = -m
         mf, nf = mgrid[0:mp+1, 0:n+1]
-        sv = errprint(0)
-        if abs(z) < 1:
-            # Ferrer function; DLMF 14.9.3
-            fixarr = where(mf > nf, 0.0,
-                           (-1)**mf * gamma(nf-mf+1) / gamma(nf+mf+1))
-        else:
-            # Match to clpmn; DLMF 14.9.13
-            fixarr = where(mf > nf, 0.0, gamma(nf-mf+1) / gamma(nf+mf+1))
-        sv = errprint(sv)
+        with ufuncs.errstate(all='ignore'):
```

Comment:
Probably not a good one. The imports at the top also import things from `_ufuncs` and I was trying to be consistent, but probably everything should just be cleaned up to use `special` instead. There are a couple of other spots like that lurking in the special tests. I wonder what the history is?